app/helpers/application_helper/button/instance_miq_request_new.rb

```diff
@@ -0,0 +1,19 @@
+class ApplicationHelper::Button::InstanceMiqRequestNew < ApplicationHelper::Button::Basic
+  def disabled?
+    !provisioning_supported?
+  end
+
+  def calculate_properties
+    super
+
+    if disabled?
+      self[:title] = N_('No Cloud Provider that supports instance provisioning added')
+    end
+  end
+
+  private
+
+  def provisioning_supported?
+    EmsCloud.all.any? { |provider| provider.supports_provisioning? }
```

Comment:
@vecerek Yup, definitely it is an improvement, and the rest we can figure out once #5597 is solved